scripts/ceph-windows/setup_ceph_vstart

```diff
@@ -51,7 +60,7 @@ mkdir -p \$HOME/ceph-vstart/out
 
 cd ~/ceph/build
 VSTART_DEST=\$HOME/ceph-vstart ../src/vstart.sh \
-    -n --memstore -o "memstore_device_bytes=$VSTART_MEMSTORE_BYTES" \
+    -n $objectstore_args \
```

Comment:
Quoting here wouldn't hurt too.

Reply:
We can't quote it here since we're passing multiple arguments, e.g. ``--bluestore -o bluestore_block_size=$VSTART_BLUESTORE_BYTES``

Reply:
Argh, yeah -- it could be done with an array but it's not worth it here.